### ISSUE TITLE
Ivy user manual (at oremacs) not clear about how to mark items in hydra menu

### ISSUE THREAD

**Issue by dalanicolai** [abo-abo/swiper#2475]
The ivy user-manual at https://oremacs.com/swiper/ states that hydra offers the possibility to mark multiple items using the key 'm'. However the key is bound to `ivy-rotate-preferred-builders` (which is also mentioned in the manual). Also 't' is bound to toggle truncate (not stated in the manual) instead of toggling marks as stated in the manual. Finally the manual states that 'u' is bound to ivy-occur while 'u' is bound to 'page up' and ivy-occur is bound to 'o'.

Anyway, thanks for the amazing extension of course!

**Comment by abo-abo** [comment]
Hi. I've looked at `ivy-hydra.el` and:

- <kbd>m</kbd> is bound to `ivy-mark` (while <kbd>M</kbd> is bound to `ivy-rotate-preferred-builders`)
- <kbd>t</kbd> is bound to `ivy-toggle-marks`
- <kbd>U</kbd> is bound to `ivy-occur`

Maybe you're using some custom config?

**Comment by dalanicolai** [comment]
Hi!
Sorry, for the false bug report. Indeed I am on spacemacs and I've tested ivy-hydra on a 'clean' spacemacs installation but not on a plain emacs installation. I tested ivy-hydra in plain emacs now and I see that indeed it works. I am new to spacemacs and emacs, so I did not expect this difference. Anyway I learned that I should check in plain emacs before sending bug reports. Thanks for looking at it and the very quick reply. I will send an 'upgrade' request to spacemacs then, and be more careful next time...
